app/src/main/java/io/github/muntashirakon/AppManager/details/AppDetailsOverlaysFragment.java

```diff
@@ -0,0 +1,263 @@
+package io.github.muntashirakon.AppManager.details;
+
+import android.content.om.IOverlayManager;
+import android.graphics.Color;
+import android.os.Build;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.annotation.RequiresApi;
+import androidx.annotation.UiThread;
+
+import com.google.android.material.card.MaterialCardView;
+import com.google.android.material.materialswitch.MaterialSwitch;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+import io.github.muntashirakon.AppManager.R;
+import io.github.muntashirakon.AppManager.compat.OverlayManagerCompact;
+import io.github.muntashirakon.AppManager.details.struct.AppDetailsItem;
+import io.github.muntashirakon.AppManager.details.struct.AppDetailsOverlayItem;
+import io.github.muntashirakon.AppManager.logs.Log;
+import io.github.muntashirakon.AppManager.self.SelfPermissions;
+import io.github.muntashirakon.AppManager.self.pref.TipsPrefs;
+import io.github.muntashirakon.AppManager.utils.ThreadUtils;
+import io.github.muntashirakon.AppManager.utils.UIUtils;
+import io.github.muntashirakon.AppManager.utils.appearance.ColorCodes;
+import io.github.muntashirakon.util.AdapterUtils;
+import io.github.muntashirakon.view.ProgressIndicatorCompat;
+import io.github.muntashirakon.widget.MaterialAlertView;
+import io.github.muntashirakon.widget.RecyclerView;
+
+public class AppDetailsOverlaysFragment extends AppDetailsFragment {
+
+    private static final String TAG = AppDetailsOverlaysFragment.class.getSimpleName();
+    private AppDetailsRecyclerAdapter mAdapter;
+    private IOverlayManager overlayManager;
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            overlayManager = OverlayManagerCompact.getOverlayManager();
+        }
+    }
+
+    @Override
+    public void onCreateMenu(@NonNull Menu menu, @NonNull MenuInflater menuInflater) {
+        menuInflater.inflate(R.menu.fragment_app_details_overlay_actions, menu);
+    }
+
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        String emptyStringText;
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
+            emptyStringText = getString(R.string.overlay_sdk_version_to_low);
+        } else if (!SelfPermissions.checkSelfOrRemotePermission("android.permission.CHANGE_OVERLAY_PACKAGES")) {
```

Comment:
Please add this permission to the `ManifestCompat` class along with `@RequiresApi`.